docs/source/GetStartedGuide.rst

```diff
@@ -111,6 +111,22 @@ having the right compilers, header files and libraries available.  The
 more predictable and complete Avocado experience can be achieved with
 the official RPM packages.
 
+Build custom docker image
+--------------------------
+
+One of possible ways to develop avocado is to create docker image with your devel tree.
+This is good way to test your devel branch w/o breaking system instalation ::
+  # Fetch git source as usual
+  $ git clone github.com/avocado-framework/avocado.git avocado.git
+  $ cd avocado.git
+  # Make some changes
+  $ patch -p1 < MY_PATCH
+  # Finally build a docker image
+  $ docker build -t fedora-avocado-custom -f examples/install/Dockerfile.fedora .
+  # Run test inside custom docker image
+  $ avocado run --docker fedora-avocado-custom examples/tests/passtest.py
```

Comment:
Sure, feel free to replace this patch with the link to your debian Dockerfile (in the `RunningTestsRemotely` section).

As for the git-based image, I'd prefer to keep it outside the main avocado tree. But if you really feel like including it, I'd use the `contrib` section as suggested by Ademar. Alternatively you can share it in your tree and only put link to the documentation.